### ISSUE TITLE
spec file unable to deal with "say_hello" method.

### ISSUE THREAD

**Issue by mikeappell** [learn-co-curriculum/jukebox-cli#21]
The instructions ask us to create a small intro program within jukebox.rb as follows:

	def say_hello(name)
	  "Hi #{name}!"
	end

	puts "Enter your name:"
	users_name = gets.chomp
	puts say_hello(users_name)

However, nitrous.io's terminal is unable to deal with 'gets' requests, and there's no stub created in the spec file to help the program deal with this if we do implement it.

It might be worth either adding something like 

	self.stub(:gets).and_return("Your Name Here")

to the spec file in the contexts under which it would use the runner, or else make it clear that this isn't necessary for the program's completion.

**Comment by SophieDeBenedetto** [comment]
Hi @mikeappell, are you running into this issue when you try to run this mini-program with `ruby lib/jukebox.rb`? Or does it happen if you try to run the test suite with the above code in `lib/jukebox.rb`? 

@jmburges is it true that Nitrous can't handle the `gets` method?

**Comment by jmburges** [comment]
@SophieDeBenedetto good catch! maybe that's what's up. @EricR any ideas?

**Comment by mikeappell** [comment]
Hi all,

I believe the issue was when running it via the test suite, and I was able
to bypass it by adding that extra stub at the beginning of any test method
which called 'run' (ended up being the last 3.)

As for Nitrous not working with 'gets', I'd seen multiple references to
that in Ask A Question answers provided to various people, most saying
something to the effect that without stubbing for the 'gets' method the
tests wouldn't work. I also remember running into that issue myself, though
I don't remember the circumstances. Perhaps it only was an issue when
running spec tests? Either way, I just ran my Jukebox program, complete
with 'gets' method, and it seemed to work fine and as expected, so I'm not
certain.
